packages/rune-games-cli/src/flows/List/Details.tsx

```diff
@@ -0,0 +1,37 @@
+import { Text, Box } from "ink"
+import React from "react"
+
+import { Step } from "../../components/Step.js"
+import { useGame } from "../../gql/useGame.js"
+import { useMe } from "../../gql/useMe.js"
+
+export function Details({ gameId }: { gameId: number }) {
+  const { me } = useMe()
+  const { game, gameLoading } = useGame(gameId)
```

Comment:
Maybe `me`, `game` and `gameLoading` should be passed in as props, since they are already fetched by the parent `List` component?

Reply:
I was considering this but then I concluded that it would be better to adjust our apollo fragments and cache policy so that `useGame` and `useMe` instantly return cached data. But sure, let's do that right away until we figure out that out.

Reply:
Done in https://github.com/rune/rune/pull/292/commits/ff22bccf453cd1423dad25e998e6b233f53f0e68